draft-ietf-intarea-provisioning-domains.md

```diff
@@ -931,13 +933,19 @@ leaking identity information, SHOULD make use of an IPv6 Privacy Address
 and SHOULD NOT include any privacy sensitive data, such as User Agent
 header or HTTP cookie, while performing the HTTP over TLS query.
 
-From a privacy perspective, retrieving the PvD Additional Information
+From a user privacy perspective, retrieving the PvD Additional Information
 is not different from establishing a first connection to a remote
 server, or even performing a single DNS lookup. For example, most
 operating systems already perform early queries to well known web sites,
 such as http://captive.example.com/hotspot-detect.html, in order to
 detect the presence of a captive portal.
 
+The DNS queries associated with the PvD Additional Information MUST
+use the DNS servers indicated by the associated PvD, as described in
+{{retr}}. This ensures the name of the PvD Additional Information server
+is not unintentionally sent on another network, thus identifying information
```

Comment:
thus *leaking* indentifying...?

Reply:
Fixed!

Reply:
s/the DNS server addresses SHOULD be associated/the recursive DNS server addresses SHOULD be associated/